tests/yam/agama/boot_agama.pm

```diff
@@ -47,6 +47,9 @@ sub prepare_boot_params {
         set_var('INST_AUTO', $profile_url);
         push @params, "inst.auto=\"$profile_url\"", "inst.finish=stop";
     }
+    if (get_var('HDD_2') =~ /oemdrv/) {
+        push @params, 'inst.finish=stop';
+    }
```

Comment:
let's do this in single line and put a comment to it on top why:
'Disk label with OEMDRV will automatically pick the profile from disk without setting inst.auto" for example.

Reply:
updated code and VR.